test/input_files/v4.2/failed/failed_body_sample_010.vcf

```diff
@@ -1,5 +1,5 @@
 ##fileformat=VCFv4.2
-##CauseOfFailure=Sample field does not match Number definition in meta section
```

Comment:
BTW, I just realised that making the error ("there are several ploidies within the same contig") a warning, those files are no longer invalid. I'll remove them